flink-libraries/flink-table/src/test/scala/org/apache/flink/table/utils/TestFilterableTableSource.scala

```diff
@@ -108,11 +108,15 @@ class TestFilterableTableSource(
   private val filterValues = new mutable.ArrayBuffer[Int]
 
   override def getDataSet(execEnv: ExecutionEnvironment): DataSet[Row] = {
-    execEnv.fromCollection[Row](applyPredicatesToRows(data).asJava, getReturnType)
+    execEnv
+      .fromCollection[Row](applyPredicatesToRows(data).asJava, getReturnType)
+      .name(explainSource())
```

Comment:
Would be great if you can be consistent with parenthesis. Sometimes it is `explainSource()` and sometimes `explainSource`.